pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1747,6 +1814,12 @@ class Build {
                         throw new Exception("[ERROR] Installer job timeout (${buildTimeouts.INSTALLER_JOBS_TIMEOUT} HOURS) has been reached OR the downstream installer job failed. Exiting...")
                     }
                 }
+                try {
+                    gpgSign()
+                    }
```

Comment:
one too many `}`?

Reply:
Yup ... Fixed. The linter inevitably picked that up too.